src/frontend/src/types/interfaces/pia-intake.interface.ts

```diff
@@ -13,7 +13,8 @@ export interface IPIAIntake {
       mpoName: string,
       mpoEmail: string,
       initiativeDescription: string,
+      initiativeScope: string,
       dataElementsInvolved: string,
-      hasAddedPiDataElements: boolean | null,
-      riskMitigation: string | null,
+      hasAddedPiToDataElements: boolean,
+      riskMitigation: string | undefined,
 }
```

Comment:
Don't these have to accept null as a potential type as we are passing them in as null in some cases?